pkg/controller/bundlec/controller.go

```diff
@@ -91,6 +98,12 @@ func (c *Controller) Run(ctx context.Context) {
 	c.Logger.Info("Starting Bundle controller")
 	defer c.Logger.Info("Shutting down Bundle controller")
 
+	sink := core_v1_client.EventSinkImpl{
+		Interface: c.MainClient.CoreV1().Events(""),
```

Comment:
nit: `meta_v1.NamespaceNone`. Hm, I'm wondering if an actual namespace should be used here. Does not look like kube is doing it consistently. Lets keep it as is.